### PR TITLE
#268 Enable creating a new committee member via GUI?

### LINKED ISSUE #108: [5] #268 Enable creating a new committee member via GUI?
Original issue https://github.com/bitshares/bitshares-ui/issues/268

### PR DESCRIPTION
**Pull request by VerevkinAlexander**
<h2>General</h2>
Closes  #108

I`ve added create a committee modal

<h2>General</h2>
Please make sure the following is done:

- Pull request is onto develop
- If you haven't already, have a look at 
  - https://github.com/bitshares/bitshares-ui/blob/develop/CODE_OF_CONDUCT.md
  - https://github.com/bitshares/bitshares-ui/blob/develop/CONTRIBUTING.md

<h2>Code Preparation</h2>

_Please review all your changes one last time before committing_

- [ ] Check for unused code
- [ ] No unrelated changes are included
- [ ] None of the changed files are reformatting only
- [ ] Code is self explanatory or documented
- [ ] All written text is properly translated (english language)

<h2>Testing</h2>

_The branch has been tested on the following browsers (desktop and mobile view)_

- [x] Chrome 
- [ ] Opera
- [ ] Firefox
- [ ] Safari

<h2>User interface changes</h2>

image

image

image

image

image

- [x] Dark
- [ ] Light
- [ ] Midnight

_Please provide screenshots/licecap of your changes below_

### CODE REVIEW

**Comment by VerevkinAlexander** [comment]
I`ve finished please review it

File: app/components/Account/AccountVoting.jsx [hunk 1]
```diff
@@ -890,6 +907,15 @@ class AccountVoting extends React.Component {
                                                 marginTop: "-2.5rem"
                                             }}
                                         >
+                                            <button
```
Comment (sschiessl-bcp) [inline app/components/Account/AccountVoting.jsx:910]: StyleGuide Button

File: app/components/Account/AccountVoting.jsx [hunk 2]
```diff
@@ -925,6 +951,11 @@ class AccountVoting extends React.Component {
                                         proxy={this.state.proxy_account_id}
                                     />
                                 </div>
+                                <JoinWitnessesModal
+                                    show={this.state.showCreateWitnessModal}
```
Comment (sschiessl-bcp) [inline app/components/Account/AccountVoting.jsx:955]: Stick to common wording and usage of modals. (`visible` e.g.)

File: app/components/Modal/JoinCommitteeModal.jsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+class JoinCommitteeModal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = this.getInitialState(props);
+    }
+
+    getInitialState() {
+        return {
+            committeeAccount: this.props.account,
+            urlSchema: "Https://"
```
Comment (sschiessl-bcp) [inline app/components/Modal/JoinCommitteeModal.jsx:17]: Why use urlSchema?

File: app/components/Modal/JoinCommitteeModal.jsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+                <Select.Option value="Https://">Https://</Select.Option>
+            </Select>
+        );
+
+        return (
+            <Modal
+                title={counterpart.translate(
+                    "modal.committee.create_committee"
+                )}
+                closable={false}
```
Comment (sschiessl-bcp) [inline app/components/Modal/JoinCommitteeModal.jsx:76]: use onCancel

**Review by sschiessl-bcp — COMMENTED** [review]
@VerevkinAlexander please have a look at my comments.

This branch is closed because I have included it in #114

File: app/components/Account/AccountVoting.jsx (hunk 1 above)
Comment (VerevkinAlexander) [reply app/components/Account/AccountVoting.jsx:910]: ok

File: app/components/Account/AccountVoting.jsx (hunk 2 above)
Comment (VerevkinAlexander) [reply app/components/Account/AccountVoting.jsx:955]: ok

File: app/components/Modal/JoinCommitteeModal.jsx (hunk 3 above)
Comment (VerevkinAlexander) [reply app/components/Modal/JoinCommitteeModal.jsx:17]: to select shemas

File: app/components/Modal/JoinCommitteeModal.jsx (hunk 4 above)
Comment (VerevkinAlexander) [reply app/components/Modal/JoinCommitteeModal.jsx:76]: ok

**Pull request closed without merging**